addon/adapters/drf.js

```diff
@@ -15,23 +15,8 @@ import Ember from 'ember';
  */
 export default DS.RESTAdapter.extend({
   defaultSerializer: "DS/djangoREST",
-
-  init: function() {
-    this._super();
-
-    if (this.get('coalesceFindRequests')) {
-      var error = "Please ensure coalesceFindRequests is not present or set " +
-        "to false in your adapter. This adapter does not support the " +
-        "coalesceFindRequests option. The Django REST Framework does not " +
-        "offer easy to configure support for N+1 query requests in the " +
-        "format that Ember Data uses (e.g. GET /comments?ids[]=1&ids[]=2) " +
-        "See the Ember documentation about coalesceFindRequests for " +
-        "information about this option: " +
-        "http://emberjs.com/api/data/classes/DS.RESTAdapter.html#property_coalesceFindRequests";
-      throw new Ember.Error(error);
-    }
-  },
```

Comment:
I will add this.  The [Ember.Logger.warn](http://emberjs.com/api/classes/Ember.Logger.html#method_warn) method will suffice.  These warnings are silenced in production I think.

I like the idea of adding an envrionment option to disable EDA logging, but let's defer that to a follow-up PR.